website_cookiefirst/views/portal_template.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <odoo>
-    <template id="footer_custom" inherit_id="website.footer_custom">
+    <template id="layout" inherit_id="portal.frontend_layout">
```

Comment:
For not losing the change when installing website, it should be:

```suggestion
    <template id="layout" inherit_id="website.layout">
```

Reply:
Ah, and better to keep the same XML-ID for avoiding problems on migrations:

```suggestion
    <template id="footer_custom" inherit_id="website.layout">
```

Reply:
if I do that I have the previus error

Reply:
Try with a less strict xpath `//footer//span...` instead of `//footer//div/span`